README.md

```diff
@@ -24,23 +24,44 @@ backpressure support.
 
 ## Features
 
-- All MQTT 3.1.1 and MQTT 5.0 features
+- **All MQTT [3.1.1](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/errata01/os/mqtt-v3.1.1-errata01-os-complete.html) and 
```

Comment:
here you exclude MQTT from the link to the 3.1.1 spec but for MQTT 5 you include MQTT in the link